bios/boot_sector/src/dap.rs

```diff
@@ -38,11 +38,11 @@ impl DiskAddressPacket {
         let self_addr = self as *const Self as u16;
         unsafe {
             asm!(
-                "push 0x7a", // error code `z`, passed to `fail` on error
+                "push 'z'", // error code `z`, passed to `fail` on error
```

Comment:
The same improvement can be done in a couple of other places.

Reply:
I only found this in one place under stage-2, where I made the same change.